packages/medusa/src/api/routes/admin/products/create-variant.ts

```diff
@@ -8,7 +8,11 @@ import {
   ValidateNested,
```

Comment:
thought (this-file): this would be a place where we would need a distributed transaction. I suggest we hold off with further work on this PR until we have an idea of how the TO will be implemented. cc: @carlos-r-l-rodrigues